File: lab/python/POO/pessoa.py
class Pessoa:
    def __init__(self, nome, idade):
        self.nome = nome
        self.idade = idade

    def apresentar(self):
        return f"Olá, meu nome é {self.nome} e eu tenho {self.idade} anos."


# Exemplo de uso
if __name__ == "__main__":
    p = Pessoa("Alice", 30)
    print(p.apresentar())

File: lab/javascript/POO/pessoa.js
class Pessoa{
    constructor(nome, idade){
        this.nome = nome;
        this.idade = idade;
    }

    apresentar(){
        console.log(`Olá, meu nome é ${this.nome} e tenho ${this.idade} anos.`);
    }
}


function Main(){
    const p = new Pessoa("João", 30);
    p.apresentar();
}

Main();
